dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -505,7 +505,7 @@ public void testUpdateUserMetadataFromGithub() {
         System.out.println(usersApi.getUser().getUserProfiles().get("github.com"));
         assertNull(userProfile.getName());
         assertEquals("dockstore.test.user2@gmail.com", userProfile.getEmail());
-        assertEquals("https://avatars1.githubusercontent.com/u/17859829?v=4", userProfile.getAvatarURL());
+        assertTrue(userProfile.getAvatarURL().endsWith("githubusercontent.com/u/17859829?v=4"));
```

Comment:
Saw your comments on Slack... Just curious what changed? It's a random digit after `avatars`, or a different one? Or the subdomain isn't there at all?

Reply:
`https://avatars1` lost the `1`